symengine/polys/uintpoly_flint.cpp

```diff
@@ -80,15 +80,17 @@ integer_class UIntPolyFlint::eval(const integer_class &x) const
 {
     flint::fmpzxx r;
     fmpz_set_mpz(r._data().inner, get_mpz_t(x));
-
-    flint::fmpzxx ans(poly_(r));
-    return to_integer_class(ans);
+    return to_integer_class(static_cast<flint::fmpzxx>(poly_(r)));
 }
 
 integer_class UIntPolyFlint::get_coeff(unsigned int x) const
 {
-    flint::fmpzxx ans(poly_.get_coeff(x));
-    return to_integer_class(ans);
+    return to_integer_class(poly_.coeff(x));
+}
+
+flint::fmpzxx_srcref UIntPolyFlint::get_coeff_ref(unsigned int x) const
```

Comment:
For now that's what is needed. We can change to `fmpzxx_ref` if it is needed later